lib/markdown.js

```diff
@@ -49,9 +49,7 @@ module.exports = function (input, options, cb) {
 
     // inject custom css if exist
     // todo: validate css
-    if (options.customCss) {
-      htmlHeader += '<link rel="stylesheet" href="' + path.resolve(options.customCss) + '">'
-    }
+    options.customCss.forEach(_ => (htmlHeader += '<link rel="stylesheet" href="' + path.resolve(_) + '">'))
```

Comment:
fair enough about it being intermediate output.  In general I like the boyscout principle to code - always leave camp clean than you found it! :) 
This is fine though, I will merge it in.